environment/src/test/java/com/sequenceiq/environment/environment/validation/EnvironmentValidatorServiceTest.java

```diff
@@ -69,9 +76,15 @@
     @Mock
     private TagValidator tagValidator;
 
+    @Mock
+    private EntitlementService entitlementService;
+
     @InjectMocks
     private EnvironmentValidatorService underTest;
 
+    @InjectMocks
```

Comment:
This should be replaced with `@Mock`. This is so that the real logic of `EncryptionKeyUrlValidator.validateEncryptionKeyUrl()` can be avoided and replaced with a mock invocation. As a consequence, you will have to add `when(encryptionKeyUrlValidator.validateEncryptionKeyUrl(...)).thenReturn(...)` expectations to the test cases.